src/tools/miri/src/shims/windows/foreign_items.rs

```diff
@@ -547,6 +547,11 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriInterpCxExt<'mir, 'tcx> {
                 // FIXME: this should return a nonzero value if this call does result in switching to another thread.
                 this.write_null(dest)?;
             }
+            "GetActiveProcessorCount" => {
+                let [groupnumber] = this.check_shim(abi, Abi::System { unwind: false }, link_name, args)?;
+                this.read_scalar(groupnumber)?.to_u16()?;
+                this.write_scalar(Scalar::from_u32(this.machine.num_cpus), dest)?;
+            }
```

Comment:
This probably wrong, need advice.

Reply:
We were already summoned by rustbot automatically :)

Why do you think this is wrong?

Reply:
This should be further up the match, down here is the section for `if this.frame_in_std()` shims. Probably putting it with GetSystemInfo makes sense.

Other than that -- without knowing GetActiveProcessorCount, this seems reasonable? A brief comment explaining why we can ignore the groupnumber might be good.

Reply:
Moved and added comment.